src/StudyViewer/reduxer.js

```diff
@@ -41,6 +41,10 @@ export const fetchFiles = (dataType, typeOfFileIndex, rowAccessorValue) => {
   }
   let nameOfIndex;
   const fieldsToFetch = ['file_name', 'file_size', 'data_format', 'data_type', targetStudyViewerConfig.rowAccessor];
+  const requiredIdpField = targetStudyViewerConfig.buttons.find((obj) => obj.type === 'request_access')?.required_idp_field;
+  if (requiredIdpField) {
+    fieldsToFetch.push(requiredIdpField);
+  }
```

Comment:
why do we need to add this to `fetchFiles`? i don't think documents in those indices will have the `required_idp` field

Reply:
we dont, i removed it, dont know why it was added. Thanks for catching it